tensorflow/compiler/tf2tensorrt/convert/utils.h

```diff
@@ -148,6 +148,16 @@ Status ContainerToTrtDims(const Container& shape, nvinfer1::Dims* trt_dims,
   return Status::OK();
 }
 
+template <typename TensorShapeType>
+nvinfer1::Dims TensorShapeToTrtDimsUnsafe(const TensorShapeType& tensor) {
```

Comment:
can we add a comment to state the assume of this routine, that is, nbDims > 0? the routine can't handle nbDims <= 0 properly, which is the reason for naming it as "Unsafe".

Reply:
Function was removed as it is now redundant DimsAdapter